onnxjs/tests/test_names.ts

```diff
@@ -0,0 +1,47 @@
+import { expect } from 'chai';
+import * as ort from 'onnxruntime-node';
+
+describe('name tests', async () => {
+    const session = await ort.InferenceSession.create('../bin/model.onnx');
+    expect(session).to.be.not.null;
+
+    const githubHandleRegex:RegExp = /\B@([a-z0-9](?:-(?=[a-z0-9])|[a-z0-9]){0,38}(?<=[a-z0-9]))/gi;
+    const backtickRegex:RegExp = /`+[^`]+`+/gi;
```

Comment:
Don't add JavaScript/TypeScript code. If you want to write more tests, just add more cases to `test_cases.json`.